src/content/transformers/aiChatPromptTransformer.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  Transformer,
+  AIChatPrompt,
+  AIChatPrompts,
+} from '../content.types';
+
+export const aiChatPromptsTransformer: Transformer<
+  AIChatPrompt,
+  AIChatPrompts
+> = (prompts: AIChatPrompt[]): AIChatPrompts[] => {
+  const aiChatPrompts = {};
+
+  for (let i = 0; i < prompts.length; i++) {
+    const prompt = prompts[i];
+    const { name } = prompt.data;
+    aiChatPrompts[name] = {
+      ...prompt.data,
+      id: prompt.id
+    }
+  }
+  return [aiChatPrompts];
+}
```

Comment:
This could be done w/ a simple `(prompts: AIChatPrompt[]): AIChatPrompts[] => prompts.map()`, no? 🤔

Reply:
I don't think so, because I'm taking all of the 'aiChatPrompt' and building one object from them all, but I went ahead and changed the loop type to (const prompt of prompts) for you to make it prettier